content/about/publications/index.md

```diff
@@ -1,6 +1,12 @@
 ---
 title: Research and White Papers
 papers:
+  - title: "Causal inference in the time of COVID-19"
+    image: causal.jpg
+    authors: Bonvini, Kennedy, Ventura, Wasserman
+    link: https://publishing.andrew.cmu.edu/collections/delphi-web/publications/causal-inference-in-the-time-of-covid-19.pdf
```

Comment:
You could also link to the preprint on arXiv: https://arxiv.org/abs/2103.04472

also is publishing.andrew.cmu.edu the final link? I've been using https://www.cmu.edu/delphi-web/

Reply:
Is the preprint link preferred? I didn't know it was on arXiv, larry and roni just sent a pdf

Reply:
Larry votes for arXiv; lemme fix